### DISCUSSION TITLE
CityTv ca

### DISCUSSION THREAD

**Discussion by clips519** [iptv-org/community#860] [Show and tell]
### Channel ID

CITYDT.ca

### Stream URL

https://www.citytv.com/video/live

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by Toei79** [comment]
Geo

**Comment by clips519** [comment]
ya ive tried to get it threw the web browser useing the inspect network but no m3u8 anyother way?

**Comment by dtngfan85** [comment]
Citytv's livestreams are DRM protected

**Comment by Toei79** [comment]
Sometimes  I get lucky with streamlink and play on tvheadend ,also you can add a free proxy to it.

**Comment by clips519** [comment]
well was worth a try thanks for the help guys only city one i could get was city news

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19843
